### ISSUE TITLE
Bulk labelling from Grid

### ISSUE THREAD

**Issue by shaileshj2803** [heartexlabs/label-studio#1667]
Allow users to select multiple items and label them all in one click. for eg Select 10 items and assign them category c1

**Comment by makseq** [comment]
Relative github issues:

https://github.com/heartexlabs/label-studio/issues/1239

Relative slack questions:

1 https://label-studio.slack.com/archives/CQ8LYPPJS/p1632468259281800?thread_ts=1632468259.281800&cid=CQ8LYPPJS

2 https://label-studio.slack.com/archives/C01Q5G6NMKR/p1621256207188200?thread_ts=1621256207.188200&cid=C01Q5G6NMKR

3 https://label-studio.slack.com/archives/CQ8LYPPJS/p1612781014326200?thread_ts=1612781014.326200&cid=CQ8LYPPJS

4 https://label-studio.slack.com/archives/CQ8LYQ0J2/p1609756561389100?thread_ts=1609756561.389100&cid=CQ8LYQ0J2

**Comment by uriel-tannen** [comment]
+1 for this feature...it would be most useful!
Another relative Slack question to add to @makseq's list:
https://label-studio.slack.com/archives/CQ8LYPPJS/p1637932002429400

**Comment by makseq** [comment]
@uriel-tannen @shaileshj2803 I've implemented some experimental functionality for this: you have to add `EXPERIMENTAL_FEATURES=true` at Label Studio startup. After this you will be able to use the 'Propagate Annotations' action in the Data Manager. Note that it can be unstable and works well only for choices (you may use it for other types of labeling at your own risk).

**Comment by uriel-tannen** [comment]
Thanks @makseq - I'll test this out. Are these experimental features in the Master branch?

**Comment by makseq** [comment]
@uriel-tannen Yes, they are in the master

**Comment by Dario-Mantegazza** [comment]
Hi, I think I might have asked a relevant question https://label-studio.slack.com/archives/C01Q5G6NMKR/p1647355469784679 
Any updates on the features? is it still experimental? 
Thanks

**Comment by makseq** [comment]
@Dario-Mantegazza Yes, it's available as experimental feature, but it's easy to enable.

**Comment by Dario-Mantegazza** [comment]
Hi @makseq, Thanks for the reply.
I tried the functionality and since all my samples are images with the same resolution and my annotation is a label for the whole image I think I'm in the "stable usage", unfortunately, I get some errors and it doesn't work. Would it be useful if I send you more information or it is an expected instability due to the experimental status of the functionality?

**Comment by makseq** [comment]
@Dario-Mantegazza Sure, please send more info us. The best is to have video with your screen here.

**Comment by AbubakarSaad** [comment]
Hello,

This feature now exist: https://docs.humansignal.com/guide/labeling_bulk.html
